lib/common/api/net-client-request.ts

```diff
@@ -288,8 +288,12 @@ function parseOptions (optionsIn: ClientRequestConstructorOptions | string): Nod
     origin: options.origin,
     referrerPolicy: options.referrerPolicy,
     cache: options.cache,
-    allowNonHttpProtocols: Object.hasOwn(options, kAllowNonHttpProtocols)
+    allowNonHttpProtocols: Object.hasOwn(options, kAllowNonHttpProtocols),
+    priority: options.priority
   };
+  if ('priorityIncremental' in options) {
+    urlLoaderOptions.priorityIncremental = options.priorityIncremental;
+  }
```

Comment:
any reason not to have this as a key in the object above unconditionally?

Reply:
To preserve the default value(true), otherwise the priorityIncremental will be false if it's assigned unconditionally to `undefined`.